verify-merge.py

```diff
@@ -9,35 +9,45 @@
 
 def verify():
     global args, workdir
-    os.chdir('gitian-pubkeys')
-    print('Importing pubkeys...')
-    keys = [f for f in glob.glob("*.asc", recursive=True)]
-    for key in keys:
-        subprocess.check_call([GPG, '--import', key])
-    print('Refreshing pubkeys...')
-    subprocess.check_call([GPG, '--refresh'])
-    os.chdir('../../gitian-builder')
-    print('\nVerifying '+args.version+' Linux\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-linux', '../monero/contrib/gitian/gitian-linux.yml'])
-    print('\nVerifying '+args.version+' Windows\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-win', '../monero/contrib/gitian/gitian-win.yml'])
-    print('\nVerifying '+args.version+' MacOS\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-osx', '../monero/contrib/gitian/gitian-osx.yml'])
+    if args.import_keys:
+        os.chdir('gitian-pubkeys')
+        print('Importing pubkeys...')
+        keys = [f for f in glob.glob("*.asc", recursive=True)]
+        for key in keys:
+            subprocess.check_call([GPG, '--import', key])
+        os.chdir('../')
+    if args.refresh_keys:
+        print('Refreshing pubkeys...')
+        subprocess.check_call([GPG, '--refresh'])
+    if not os.path.isdir(args.gitian_builder_dir):
+        print('Please clone the gitian-builder repository from github.com/devrandom/gitian-builder to the directory containg the gitian.sigs repository.\n If you already have the gitian.sigs directory cloned, but under another name, or path, use --gitian-builder-dir to pass its absolute directory path to the script.')
```

Comment:
Errors should be printed to stderr, eg:
```python
sys.stderr.write('Please clone the gitian-builder repository from github.com/devrandom/gitian-builder to the directory containing the gitian.sigs repository.\nIf you already have the gitian.sigs directory cloned, but under another name, or path, use --gitian-builder-dir to pass its absolute directory path to the script.\n')
```
(Also removed 1 leading whitespace after `\n` and added a `\n` at the end of the message to return prompt to its own line on exit)

Reply:
After the error message is printed, the script should exit to prevent the program from continuing and dumping a traceback:
```python
sys.exit(1)
```

Reply:
Done.